helpers.go

```diff
@@ -139,3 +141,32 @@ func decodeSlice(elemType reflect.Type, slice reflect.Value, input map[string]in
 
 	return nil
 }
+
+// customDecodeHook supports decoding to time
+func customDecodeHook(from reflect.Type, to reflect.Type, data interface{}) (interface{}, error) {
```

Comment:
If this is only for time better to name it as such so its clear what it does e.g. timeHookFunc